db/app.js

```diff
@@ -13,14 +13,18 @@ const {
 
 const {
   getCommentsByArticleId,
+  postCommentByArticleId,
 } = require('../controllers/comments.controllers');
 
 const {
   customErrorHandler,
+  psqlErrorHandler,
   serverErrorHandler,
   notFoundErrorHandler,
 } = require('../db/errors/index');
 
+app.use(express.json());
```

Comment:
Really good demonstration of TDD principles to only include this now that we ___need___ it to pass our test, fantastic job! 🥇

Reply:
Thanks! I still spent 5 full minutes trying to work out why I couldn't destructure my request body before adding it in, but every day is a learning experience :face_holding_back_tears: